pages.es/common/git-rev-list.md

```diff
@@ -9,7 +9,7 @@
 
 - Muestra los commits más reciente a partir de una fecha y una rama específica:
```

Comment:
This is not strictly related to this PR, but while we're at it, we should fix the typo:
```suggestion
- Muestra los commits más recientes a partir de una fecha y una rama específica:
```

@navarroaxel can you confirm please?

Reply:
Yep, it's good.